src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.spring.service.impl;
+
+import java.util.List;
+import mate.academy.spring.dao.UserDao;
+import mate.academy.spring.exception.UserNotSavedException;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+    private final UserDao userDao;
+
+    @Autowired
+    public UserServiceImpl(UserDao userDao) {
+        this.userDao = userDao;
+    }
+
+    @Override
+    public User add(User user) throws UserNotSavedException {
```

Comment:
```suggestion
    public User add(User user) {
```
let's remove this from all layers